docs/release-notes/index.md

```diff
@@ -16,6 +16,7 @@ To check for security updates, go to [Security announcements for the Elastic sta
 % ### Features and enhancements [beats-versionext-features-enhancements]
 
 % ### Fixes [beats-versionext-fixes]
+* Filebeat Journald input now works on Docker containers [#41278]({{beats-issue}}41278) [#44040]({{beats-issue}}44040) [#44056]({{beats-pull}}44056)
```

Comment:
Are we supposed to put this release notes here manually now? I must have missed a communication about it.

Reply:
Honestly, I'm still a bit confused about release notes with the doc changes. My understanding is that we should use the markdown files now.

Reply:
I think we could also put it in both places for now, just to be safe.

Reply:
I'll confirm with the docs team and, then I'll move it to the correct place and try documenting it as well.